### DISCUSSION TITLE
DITA OT 3.5.4- CHM output issue

### DISCUSSION THREAD

**Discussion by MinalNish** [dita-ot/dita-ot#4062] [Q&A]
Hi all,
I am getting the script error in the CHM output file generated using DITA OT 3.5.4. I have tried disabling it from the internet options but still is displays the script error message in the CHM file. Please help.

**Comment by raducoravu** [comment]
Could you maybe attach a screenshot with your error to the thread? What version of Windows are you using? Do you have the HTML Help Workshop installed?

**Reply by MinalNish** [reply]
Here is the screen capture of the error message:
<img width="338" alt="Error" src="https://user-images.githubusercontent.com/118829171/203484213-6e60c27b-ffe2-4ccd-8bc7-2f8f334c6ddc.png">

Using the Windows Version: Windows 10 Enterprise.

Could you please let me know what is HTML workshop? I am not aware of it.

**Reply by raducoravu** [reply]
HTML Help Workshop is the tool used to compile the CHM, as you obtain the CHM this means you have it installed.
Unfortunately I do not know what this error means, if you create the CHM for a very small DITA Map with 1-2 topics just to test, then open it do you get the same problem reported?

**Reply by MinalNish** [reply]
yes, I get the same error for each topic, if I create the CHM for a very small DITA Map with 1-2 topics.

**Reply by raducoravu** [reply]
Not sure what the problem is, did you customize the CHM publishing in any way? Maybe add extra javascript content in the generated HTML files? You can use a free utility like 7-Zip to expand the CHM file as an archive and look inside one of the HTML files at that line and column specified by the error message, maybe you can find the cause.
Other than that I cannot help further.

**Reply by MinalNish** [reply]
Thanks for your input. Inside HTML file I could find references to the following two javascript files which are not added by us. These are part of DITA OT plugin.
![image](https://user-images.githubusercontent.com/118829171/203516141-484bd633-1100-406b-bf37-52cae7811942.png)

**Reply by raducoravu** [reply]
If you download a DITA Open Toolkit distribution from the DITA Open Toolkit web site, and you perform a search in files looking for "nw.js" or "xquery.js" I do not think you will find any plugins bundled with the DITA OT by default containing or referencing these Javascript libraries. So maybe you have extra custom plugins in your DITA OT which add these library references to all generated HTML files.

**Reply by MinalNish** [reply]
Thank you so much for your help! we are trying to reach out to the team who has customized CHM publishing for us to check if these JS libraries were added by them or not.
